docs/governance/community-groups.md

```diff
@@ -22,6 +22,45 @@ Non-technical issues or high-level design decisions may be escalated to the [SC]
+4. Group Members: Contributors who actively engage in the group’s work, from submitting work to reviewing PRs.
+
+The working group charter must encapsulate the members’ details and their respective roles. Newcomers desiring to join should propose their inclusion via a PR to the group members list.
+
+Regular audits of participation are vital. Inactive members should be pruned from the members list to reflect current engagement levels accurately, facilitating more strategic planning of group tasks.
+
+### Working Group Meetings
+
+FINOS schedules and hosts monthly meetings for each working group to review accomplishments from the previous period and outline future work directions. Attendance is mandatory for at least the working group lead and maintainers. In case of their absence, the lead must delegate a representative to attend. To maximize meeting efficiency, the working group should prepare an agenda in advance and share it with the FINOS host.
+
+It’s crucial for key stakeholders from interdependent working groups to participate, fostering a synchronized effort and understanding of each group’s progress and trajectory. These sessions are perfect for discussing integration and potential obstacles.
+
+Additionally, a monthly All Hands meeting is convened with delegates from all groups. This platform sets the overall project course and addresses significant points from recent [SC] meetings, with each group reporting on their progress.
+
+Beyond these, working groups are encouraged to convene at least once monthly for in-depth discussions and planning, led by the group’s lead/maintainers rather than FINOS.
+
+Informal, ad hoc gatherings among group members are recommended to promote open dialogue and collaboration. While minutes aren’t mandatory, documenting and sharing outcomes is essential for transparency and continuity.
+
+Reviews of the meeting frequency and times should be reviewed at least twice annually to ensure that the meetings best serve the active members of the working group.
+
+### Working Group Charter
+
+Every working group is required to develop a charter that clearly outlines its scope and objectives. The charter should serve as a comprehensive guide that includes:
+
+* Membership and Roles: A list of all working group members, along with their specific roles and responsibilities.
+* Meeting Schedule and Records: Information on the regular meeting schedule, accompanied by a link to access the minutes of these meetings for transparency and reference.
+* Group Dependencies: The charter must also map out the interdependencies between different working groups, both upstream and downstream. It should provide links to the respective charters of these groups to streamline communication and collaboration.
+
 ## Changes to the Community Structure
 
 Changes to the community structure can be proposed through pull requests. All changes must be approved in accordance with the relevant group charter.
```

Comment:
I also think we should have an approval count before a PR is considered ready for merging.

Reply:
Got point. I have added this item.